proto/secret/compute/v1beta1/query.proto

```diff
@@ -34,10 +34,24 @@ service Query {
     rpc Codes (google.protobuf.Empty) returns (QueryCodesResponse) {
         option (google.api.http).get = "/compute/v1beta1/code";
     }
+    // Query contract hash by address
+    rpc ContractHash (QueryContractHashRequest) returns (QueryContractHashResponse) {
+        option (google.api.http).get = "/compute/v1beta1/hash/address/{address}";
+    }
     // Query contract hash by code id
     rpc ContractHashByID (QueryContractHashByIDRequest) returns (QueryContractHashResponse) {
         option (google.api.http).get = "/compute/v1beta1/hash/id/{code_id}";
     }
+    // Query contract key by address
+    rpc ContractKey(QueryContractKeyRequest) returns (QueryContractKeyResponse) {
+        option (google.api.http).get = "/compute/v1beta1/key/{address}";
+    }
+    // Query contract address by label
+    rpc AddressByLabel(QueryContractAddressByLabelRequest) returns (QueryContractAddressByLabelResponse) {
```

Comment:
Should also add the other way around

Reply:
Working on it in another PR, this PR focus on those APIs that were implemented in the queried but weren't exported by the pb